src/transaction/log_manager.c

```diff
@@ -12792,7 +12794,17 @@ cdc_make_dml_loginfo (THREAD_ENTRY * thread_p, int trid, char *user, CDC_DML_TYP
 	}
     }
 
-  loginfo_buf = (char *) malloc (record_length * 5 + MAX_ALIGNMENT);
+  /* metadata for CDC loginfo :
+   * loginfo length (int) + trid (int) + user name (32) + data item type (int) + dml_type (int) + classoid (int64)
+   * + number of changed column (int) + changed column index (int * number of column)
+   * + number of condition column + condition column index (int * number of column) */
+
+  metadata_length = OR_INT_SIZE + OR_INT_SIZE + DB_MAX_USER_LENGTH + OR_INT_SIZE + OR_INT_SIZE + OR_BIGINT_SIZE +
```

Comment:
I sometimes saw the size aligned. 
Is `metadata_length` not needed to be aligned?
(I just ask because I do not understand CDC overall)

Reply:
I saw in `teams`  replies.